reproducibility_project/src/engines/mcccs/project.py

```diff
@@ -23,40 +23,77 @@ def __init__(self):
 ex = Project.make_group(name="ex")
 
 
+def mc3s_exec():
+    """Return the path of MCCCS-MN executable."""
+    return "/home/rs/group-code/MCCCS-MN-7-20/exe-8-20/src/topmon"
+
+
 def get_system(job):
     """Return the system (mbuild filled_box) for a particular job."""
-    molecule = job.sp.molecule
-    import warnings
+    import sys
 
-    import mbuild as mb
+    import foyer
+    from constrainmol import ConstrainedMolecule
 
-    from reproducibility_project.src.utils.forcefields import load_ff
+    sys.path.append(Project().root_directory())
+    from src.utils.forcefields import load_ff
 
-    warnings.simplefilter("ignore")
-    methane = mb.Compound(name="MET")
-    methane_child_bead = mb.Compound(name="_CH4")
-    methane.add(methane_child_bead, inherit_periodicity=False)
-    box_liq = mb.fill_box(
-        compound=methane, n_compounds=1230, box=[4.5, 4.5, 4.5]
-    )
+    sys.path.append(Project().root_directory() + "/src/molecules/")
+    sys.path.append(Project().root_directory() + "/..")
+    from system_builder import construct_system
+
+    system = construct_system(job.sp)
+    molecule = get_molecules(job)[0]
```

Comment:
I am trying to understand what your workflow is here: it looks like sometimes in `construct_system` you want to build a system with the constrained molecules--is that correct? If that's the case, let's add this functionality to the `construct_system` function just in case other folks who may want a constrained molecule system can also use it. :)